components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -251,6 +251,7 @@ be lost if SCREAM_HACK_XML is not enabled.
     <!-- MAM4xx-ACI -->
     <mam4_aci inherit="atm_proc_base">
       <wsubmin type="real" doc="Minimum diagnostic sub-grid vertical velocity">0.001</wsubmin>
+      <enable_aero_vertical_mix type="logical" doc="Enable vertical mixing of interstitial aerosols and liquid number during activation">true</enable_aero_vertical_mix>
```

Comment:
Since this flag is to control the vertical mixing calculation of both aerosols and cloud droplet (?) in aci, it might be better change it to "enable_aci_vert_mix". Do we have a similar flag to control the vertical mixing calculation in SHOC?

Reply:
That's a good suggestion. Thanks. I will revise the flag name. 

@tcclevenger is working on a method to disable the vertical mixing in SHOC. Once that is ready, we will have two ways to control vertical mixing.